src/diffusers/loaders.py

```diff
@@ -85,12 +85,16 @@ def __init__(self, regular_linear_layer, lora_scale=1, network_alpha=None, rank=
 
         self.lora_scale = lora_scale
 
+    # overwrite PyTorch's `state_dict` to be sure that only the 'regular_linear_layer' weights are saved
+    # when saving the whole text encoder model
+    def state_dict(self, *args, destination=None, prefix='', keep_vars=False):
+        return self.regular_linear_layer.state_dict(*args, destination=destination, prefix=prefix, keep_vars=keep_vars)
+
     def _fuse_lora(self):
         if self.lora_linear_layer is None:
             return
 
         dtype, device = self.regular_linear_layer.weight.data.dtype, self.regular_linear_layer.weight.data.device
-        logger.info(f"Fusing LoRA weights for {self.__class__}")
```

Comment:
The problem with "info" here is that it would pop up 100+ times for models like SDXL which makes it unreadable. Think we should better remove it.